common/views/components/InfoBanner/InfoBanner.tsx

```diff
@@ -1,85 +1,94 @@
-import React, { FunctionComponent, useEffect, useState } from 'react';
+import React, {
+  useContext,
+  FunctionComponent,
+  useEffect,
+  useState,
+} from 'react';
 import { HTMLString } from '../../../../common/services/prismic/types';
 import cookie from 'cookie-cutter';
 import { grid, font } from '../../../utils/classnames';
 import Icon from '../Icon/Icon';
 import PrismicHtmlBlock from '../PrismicHtmlBlock/PrismicHtmlBlock';
 import Space from '../styled/Space';
+import GlobalInfoBarContext from '@weco/common/views/components/GlobalInfoBarContext/GlobalInfoBarContext';
 
 type Props = {
   cookieName?: string;
   text: HTMLString;
-  initialIsVisible?: boolean;
-};
-
-type State = {
-  showInfoBanner: boolean;
 };
 
 const InfoBanner: FunctionComponent<Props> = ({ cookieName, text }: Props) => {
-  const [isVisible, setIsVisible] = useState<boolean>(false);
+  const { showInfoBarBanner, setInfoBarBanner } = useContext(
+    GlobalInfoBarContext
+  );
+  useEffect(() => {
```

Comment:
it looks like we've pulled in some changes that were made [here](https://github.com/wellcomecollection/wellcomecollection.org/commit/c023b81d4a81f56f10080cfb9cb608e040a7789a#diff-6d893f831dfaa9863006df4faf1b11600e85441c6463a594717d543f2e449183).